content/lxd/getting-started-cli.md

```diff
@@ -147,10 +151,15 @@ Run the following command to specify the configuration options:
 
     sudo lxd init
 
-If you want to use the default options, add the `--auto` flag:
+**Quick test setup**
+
+To create a non-optimized test setup with default options, you can skip the configuration steps by adding the `--auto` flag:
```

Comment:
```suggestion
To create a non-optimised test setup with default options, you can skip the configuration steps by adding the `--auto` flag:
```

Reply:
`optimized` is the american english variant, so it is not incorrect, according to some dictionaries.

But if the whole project prefers to be in british english, thats up to you.

Reply:
Yes, the docs are supposed to be in British English. :)

Reply:
Interesting. I guess I might add that to the [CONTRIBUTING.md](https://github.com/lxc/linuxcontainers.org/blob/master/CONTRIBUTING.md).

Reply:
Hmm, actually, I'd personally prefer US English as that's the locale that's used in the CLI (see `lxc export --help` where we have a case of `optimized-storage` and not `optimised-storage`).

I know that as a company, our official documents we put out under the company name, like whitepapers and the like are expected to be UK English and it's fine for those, but extending that to our docs would cause them to be at odds with the actual CLI and API :)

Reply:
(And I suspect we defaulted to US spelling despite none of us being from the US, due to habit of dealing with documentation for Linux and some of our other major dependencies which all tend to default to US spelling)

Reply:
I agree that American english is probably the standard in the linux world as well.

So should I use American English instead? Or do want to discuss this further?

Reply:
yes, please